ext/GeometryOpsLibGEOSExt/buffer.jl

```diff
@@ -0,0 +1,29 @@
+const _GEOS_CAPSTYLE_LOOKUP = Dict{Symbol, LG.GEOSBufCapStyles}(
+    :round => LG.GEOSBUF_CAP_ROUND,
+    :flat => LG.GEOSBUF_CAP_FLAT,
+    :square => LG.GEOSBUF_CAP_SQUARE,
+)
+
+const _GEOS_JOINSTYLE_LOOKUP = Dict{Symbol, LG.GEOSBufJoinStyles}(
+    :round => LG.GEOSBUF_JOIN_ROUND,
+    :mitre => LG.GEOSBUF_JOIN_MITRE,
+    :bevel => LG.GEOSBUF_JOIN_BEVEL,
+)
+
+to_cap_style(style::Symbol) = _GEOS_CAPSTYLE_LOOKUP[style]
```

Comment:
```suggestion
_to_cap_style(style::Symbol) = _GEOS_CAPSTYLE_LOOKUP[style]
```

Nice to mark these little helpers as internals

Reply:
It's not accessible in any case unless you use something like `Base.get_extension(GeometryOps, :GeometryOpsLibGEOSExt)` - so is that necessary in this case?

Reply:
Its more about signalling to the reader what the scope is. Its hard to know if there was an import somewhere just looking at these lines.